### PR TITLE
Issue#207 Requests to KsqlDb from UI

### PR DESCRIPTION
**Pull request by abelgibayev**
**What changes did you make?** (Give an overview)
- Added KsqlDb page
- Added ability to make ksql queries
**Is there anything you'd like reviewers to focus on?**

**How Has This Been Tested?** (put an "X" next to an item)

- [ ] No need to
- [ ] Manually(please, describe, when necessary)
- [ ] Unit checks
- [ ] Integration checks
- [ ] Covered by existing automation

**Checklist** (put an "X" next to an item, otherwise PR will fail)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [x] My changes generate no new warnings(e.g. Sonar is happy)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged

Check out [Contributing](https://github.com/provectus/kafka-ui/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/provectus/kafka-ui/blob/master/CODE-OF-CONDUCT.md)

### CODE REVIEW

File: kafka-ui-react-app/src/components/KsqlDb/List/List.tsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+  { Header: 'Type', accessor: 'type' },
+  { Header: 'Name', accessor: 'name' },
+  { Header: 'Topic', accessor: 'topic' },
+  { Header: 'Key Format', accessor: 'keyFormat' },
+  { Header: 'Value Format', accessor: 'valueFormat' },
+];
+
+const List: FC = () => {
+  const dispatch = useDispatch();
+  const [isModalShown, setIsShow] = useState(false);
```
Comment (workshur) [inline kafka-ui-react-app/src/components/KsqlDb/List/List.tsx:21]:
```suggestion
  const [isModalShown, setIsModalShown] = useState(false);
```

Minor. just a standard naming convention

File: kafka-ui-react-app/src/components/KsqlDb/List/List.tsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+                </tr>
+              </thead>
+              <tbody>
+                {rows.map((row) => (
+                  <tr key={row.name}>
+                    {headers.map(({ accessor }) => (
+                      <td key={accessor}>{row[accessor]}</td>
+                    ))}
+                  </tr>
+                ))}
```
Comment (workshur) [inline kafka-ui-react-app/src/components/KsqlDb/List/List.tsx:90]: Add a placeholder for case when there are no rows.

File: kafka-ui-react-app/src/components/common/SQLEditor/SQLEditor.tsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+            : '500px'
+        }
+        wrapEnabled
+        {...rest}
+      />
+    );
+  }
+);
+
+SQLEditor.displayName = 'JSONEditor';
```
Comment (workshur) [inline kafka-ui-react-app/src/components/common/SQLEditor/SQLEditor.tsx:34]:
```suggestion
SQLEditor.displayName = 'SQLEditor';
```

File: kafka-ui-react-app/src/components/common/SQLEditor/__tests__/SQLEditor.spec.tsx [hunk 4]
```diff
@@ -0,0 +1,20 @@
+import { shallow } from 'enzyme';
+import React from 'react';
+import SQLEditor from 'components/common/SQLEditor/SQLEditor';
+
+describe('JSONEditor component', () => {
```
Comment (workshur) [inline kafka-ui-react-app/src/components/common/SQLEditor/__tests__/SQLEditor.spec.tsx:5]:
```suggestion
describe('SQLEditor component', () => {
```

**Review by workshur — COMMENTED** [review]
Great job @abelgibayev! 

Please add some screenshots and check SonarCloud message. It would be great if we can avoid adding code smells and Increase coverage up to 80%

File: kafka-ui-react-app/src/components/KsqlDb/QueryModal/QueryModal.tsx [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+
+  const {
+    handleSubmit,
+    control,
+    formState: { isSubmitting },
+  } = useForm<FormValues>({
+    mode: 'onTouched',
+    resolver: yupResolver(validationSchema),
+    defaultValues: {
+      ksql: '',
```
Comment (burusha16) [inline kafka-ui-react-app/src/components/KsqlDb/QueryModal/QueryModal.tsx:48]: there is second non required param `additionalProperties`.  Don't forget to add it.

**Review by burusha16 — CHANGES_REQUESTED** [review]
Can we discuss ui/ux with @workshur and @germanosin? I think we have wrong ui/ux for this feature.

**Review by workshur — APPROVED** [review]

File: kafka-ui-react-app/src/components/common/Tabs/Tabs.tsx [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+            </li>
+          ))}
+        </ul>
+      </div>
+      {React.Children.toArray(children)[selectedIndex]}
+    </>
+  );
+};
+
+Tabs.displayName = 'Tabs';
```
Comment (burusha16) [inline kafka-ui-react-app/src/components/common/Tabs/Tabs.tsx:55]: nit: same with display name

**Review by burusha16 — APPROVED** [review]

**Comment by burusha16** [comment]
🔥🔥🔥

**Pull request merged**